jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/insights/QueryEditor.java

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.cloudwatch.logs.insights;
+
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.ui.SimpleToolWindowPanel;
+import org.jdesktop.swingx.JXDatePicker;
```

Comment:
not sure how i feel about this import, but the only other reference i can find to date pickers in the IJ codebase is `com.michaelbaranov.microba.calendar.DatePicker` in the VCS integration packages